PoissonRegression.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>There should be only one output per code block -- either print the variances or plot the figure, but not both. These need to be broken into two code blocks. </p><p><br></p><p>You might also consider just running `describe(chains[201:end, :, :])`, which presents standard deviation. </p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Separated the two code blocks. Replaced the printing part by describe(chains[401:end,:,:]). I used 401 instead of 201 as I found that one of the chains was taking a bit longer to converge.</p>

Reply:
I wouldn't --- use the whole chain starting from 201. Usually we only cut out at 201 to remove the adaptation samples. It seems a little disingenuous to remove an extra 200 samples because the chain lagged a little bit.

Reply:
Okay, reverted back to cutting out at 201 samples.